python/ray/rllib/envs.py

```diff
@@ -5,8 +5,11 @@
 import gym
 import logging
 import time
+import numpy as np
+import tensorflow as tf
 
 from ray.rllib.models import ModelCatalog
+from ray.rllib.utils.overrides import overrides
```

Comment:
Hm you don't seem to need this, so it would be nice to get rid of the extra dependency.